lib/src/components/SimpleComponents.tsx

```diff
@@ -19,10 +21,10 @@ export const NxThreatNumber = withClass('span', 'nx-threat-number');
 export const NxPageMain = withClass('main', 'nx-page-main');
 export const NxPageSidebar = withClass('aside', 'nx-page-sidebar');
 
-export const NxH1 = withClass('h1', 'nx-h1');
-export const NxH2 = withClass('h2', 'nx-h2');
-export const NxH3 = withClass('h3', 'nx-h3');
-export const NxH4 = withClass('h4', 'nx-h4');
+export const NxH1 = withClass('h1', 'nx-h1', undefined, <NxSkeletonLoader.Text />);
```

Comment:
> are there cases in product that H1s need a skeleton loader?

it might as well have one since all the others do.  Going out of our way not to give it one seems like it adds needless risk in case one was ever needed for some reason.